index.html

```diff
@@ -552,39 +563,150 @@ <h3>BitstringStatusListEntry</h3>
 includes the `BitstringStatusListCredential` value.
               </td>
             </tr>
+            <tr>
+              <td id="statusSize">statusSize</td>
+              <td>
+The `statusSize` indicates the size of the status entry in bits. `statusSize`
+MAY be provided. If `statusSize` is not present as a property of the
+`credentialStatus`, then `statusSize` MUST be processed as `1`.  `statusSize`
+MUST be an integer greater than zero. If `statusSize` is provided and is greater
+than `1`, then the property `credentialStatus.statusMessage` MUST be present,
+and the number of status messages must equal the number of possible values.
+              </td>
+            </tr>
+            <tr>
+              <td id="statusMessage">statusMessage</td>
+              <td>
+The `statusMessage` property MUST be an array. If present,
+the length of the array must equal the number of possible status states
+indicated by `statusSize`. `statusMessages` MAY be present if
+`statusSize` is `1`. `statusMessages` MUST be present if
+`statusSize` is greater than `1`.  If not present, the message value
+associated with the bit value of `0` is "unset" and the bit
+value of `1` is "set".
+If present, elements in the `statusMessage` array MUST contain at
+minimum two properties:
```

Comment:
> (i.e., the `statusMessage` array MUST have as many elements as there are bits in `statusSize`).

Hmm, I would think it needs to have as many elements as there are possible values instead, right? For example, if there are 2 bits, that's 4 possible values -- so 4 elements are required, not 2.

Reply:
Ah, yes, good catch! Fixing...

Reply:
@dlongley — Not a trivial reword. See if this makes sense —
```suggestion
If present, the `statusMessage` property MUST be an array, the length
of which MUST equal the number of possible status states indicated by
`statusSize` (e.g., `statusMessage` array MUST have 2 elements if
`statusSize` has 1 bit, 4 elements if `statusSize` has 2 bits, 8
elements if `statusSize` has 3 bits, etc.). `statusMessage` MAY be
present if `statusSize` is `1`, and MUST be present if `statusSize` is
greater than `1`.  If the `statusMessage` array is not present, the
message values associated with the `status` bit values of `1` and `0`
are "set" and "unset", respectively. If the `statusMessage` array is
present, each element MUST contain the two properties described below,
and MAY contain additional properties.
```

Reply:
> "possible status messages"? Since this is for the "message" status purpose type?

The above, applied below. 

I don't have any solution bound thoughts on the other points raised [here](https://github.com/w3c/vc-bitstring-status-list/pull/164#discussion_r1558312457) (which maybe should be raised in a new, independent issue?)
```suggestion
If present, the `statusMessage` property MUST be an array, the length
of which MUST equal the number of possible status messages indicated
by `statusSize` (e.g., `statusMessage` array MUST have 2 elements if
`statusSize` has 1 bit, 4 elements if `statusSize` has 2 bits, 8
elements if `statusSize` has 3 bits, etc.). `statusMessage` MAY be
present if `statusSize` is `1`, and MUST be present if `statusSize` is
greater than `1`.  If the `statusMessage` array is not present, the
message values associated with the `status` bit values of `1` and `0`
are "set" and "unset", respectively. If the `statusMessage` array is
present, each element MUST contain the two properties described below,
and MAY contain additional properties.
```